testing/integration/enroll_test.go

```diff
@@ -31,33 +28,17 @@ func TestEnrollAndLog(t *testing.T) {
 		Sudo:  true,
 	})
 	t.Logf("got namespace: %s", info.Namespace)
-	suite.Run(t, &EnrollRunner{requirementsInfo: info})
-}
 
-type EnrollRunner struct {
-	suite.Suite
-	requirementsInfo *define.Info
-	agentFixture     *atesting.Fixture
-}
-
-func (runner *EnrollRunner) SetupSuite() {
-	runner.T().Logf("In SetupSuite")
-	agentFixture, err := define.NewFixture(runner.T(), define.Version())
-	runner.agentFixture = agentFixture
-	require.NoError(runner.T(), err)
-}
+	t.Logf("In SetupSuite")
+	agentFixture, err := define.NewFixture(t, define.Version())
+	require.NoError(t, err)
 
-func (runner *EnrollRunner) SetupTest() {
-
-}
-
-func (runner *EnrollRunner) TestEnroll() {
-	runner.T().Logf("In TestEnroll")
-	kibClient := runner.requirementsInfo.KibanaClient
+	t.Logf("In TestEnroll")
+	kibClient := info.KibanaClient
 	// Enroll agent in Fleet with a test policy
 	createPolicyReq := kibana.AgentPolicy{
 		Name:        fmt.Sprintf("test-policy-enroll-%d", time.Now().Unix()),
-		Namespace:   "enrolltest",
+		Namespace:   info.Namespace,
```

Comment:
This change is not strictly a refactoring change.  However, being a one-line change, it was convenient to include it in this PR.